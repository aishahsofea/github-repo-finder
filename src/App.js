import React from "react";
import { RepoFinder } from "./RepoFinder";
import Typography from "@material-ui/core/Typography";
import Grid from "@material-ui/core/Grid";

export class App extends React.Component {
  render() {
    return (
      <div className="App" style={{ textAlign: "center" }}>
        <Typography variant="h3">Github Repo Finder</Typography>
        <Typography variant="h6">
          Search for any topics you are interested in
        </Typography>
        <Grid container alignItems="center" justify="center">
          <RepoFinder />
        </Grid>
      </div>
    );
  }
}
