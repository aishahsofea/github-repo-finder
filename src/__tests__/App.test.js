import React from "react";
import { App } from "../App";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("<App />", () => {
  it("must render correctly", () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
