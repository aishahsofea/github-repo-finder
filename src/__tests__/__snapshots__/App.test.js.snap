// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> must render correctly 1`] = `ShallowWrapper {}`;
